pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go

```diff
@@ -68,7 +68,6 @@ func (pl *NodeUnschedulable) Filter(ctx context.Context, _ *framework.CycleState
 		Key:    v1.TaintNodeUnschedulable,
 		Effect: v1.TaintEffectNoSchedule,
 	})
-	// TODO (k82cn): deprecates `node.Spec.Unschedulable` in 1.13.
```

Comment:
iirc, this was deferred for some reasons, I am not sure remove this is the right thing to do.

Reply:
Some comments before https://github.com/kubernetes/kubernetes/pull/84266#issuecomment-545951881